### PR TITLE
fix(terraform): handle entity_configs that miss paths from definition_paths

### PR DESCRIPTION
**Pull request by arielkru**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

[//]: # "
    # PR Title
    Be aware that we use the title to create changelog automatically and therefore only allow specific prefixes
    - break:    to indicate a breaking change, this supersedes any of the types
    - feat:     to indicate new features or checks
    - fix:      to indicate a bugfix or handling of edge cases of existing checks
    - docs:     to indicate an update to our documentation
    - chore:    to indicate adjustments to workflow files or dependency updates
    - platform: to indicate a change needed for the platform
    Additionally a scope is needs to be added to the prefix, which indicates the targeted framework, in doubt choose 'general'. 
    # Allowed prefixs: argo|arm|azure|bicep|bitbucket|circleci|cloudformation|dockerfile|github|gha|gitlab|helm|kubernetes|kustomize|openapi|sca|secrets|serverless|terraform|general|graph|terraform_plan    
"

## Description

This change is supposed to avoid errors in case of broken entity configs / definition paths (based on a real case)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my feature, policy, or fix is effective and works
- [ ] New and existing tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: checkov/terraform/runner.py [hunk 1]
```diff
@@ -631,5 +631,5 @@ def get_graph_resource_entity_config(self, entity, entity_context):
         definition_path = entity_context.get('definition_path', [])
         entity_config = entity['config_']
         for path in definition_path:
-            entity_config = entity_config[path]
+            entity_config = entity_config.get(path, entity_config)
```
Comment (achiar99) [inline checkov/terraform/runner.py:634]: @arielkru i think we need to understand why this key doesnt exist. if its not related to secrets ill take this

File: checkov/terraform/runner.py [hunk 2]
```diff
@@ -629,7 +629,10 @@ def extract_images(
 
     def get_graph_resource_entity_config(self, entity, entity_context):
         definition_path = entity_context.get('definition_path', [])
-        entity_config = entity['config_']
+        entity_config = entity.get('config_', {})
```
Comment (gruebel) [inline checkov/terraform/runner.py:632]:
```suggestion
        entity_config = entity.get(CustomAttributes.CONFIG, {})
```

File: checkov/terraform/runner.py (hunk 1 above)
Comment (gruebel) [reply checkov/terraform/runner.py]: also curious about it, seems more like a bug

**Comment by arielkru** [comment]
Fixed by @YaaraVerner 's https://github.com/bridgecrewio/checkov/pull/4146

**Pull request closed without merging**